package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "Vito learn react",
+  "name": "vito-learn-react",
+  "version": "0.0.1",
+  "license": "MIT",
+  "author": "Massimo Ruggirello",
+  "contributors": [
+    "Vito Mannina"
+  ],
+  "dependencies": {
+    "bootstrap": "4.1.3"
```

Comment:
Hi @m-otiv why we need bootstrap for a playground ?

Reply:
bootstrap is evil 😈